plugins/lsp/lsp-status.nix

```
@@ -0,0 +1,57 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "lsp-status";
+  originalName = "lsp-status.nvim";
+  defaultPackage = pkgs.vimPlugins.lsp-status-nvim;
+  maintainers = [ helpers.maintainers.b3nb5n ];
+  settingsOptions =
+    let
+      indicator_opt = helpers.defaultNullOpts.mkStr "" "string to show as diagnostics. If you don't have Nerd/Awesome Fonts you can replace defaults with ASCII chars.";
```

```suggestion
      indicator_opt = helpers.defaultNullOpts.mkStr "" ''
        string to show as diagnostics.
        If you don't have Nerd/Awesome Fonts you can replace defaults with ASCII chars.
      '';
```

Start each sentence with a capital letter.
Use 2 single quotes for descriptions, with a line break.

---

Also, don't be afraid of longer multi-line descriptions. We often include examples or longer explanations this way.

Markdown syntax and code blocks are also supported.